sql/database.sql

```diff
@@ -10900,3 +10521,533 @@ CREATE TABLE IF NOT EXISTS `medex_recalls` (
   PRIMARY KEY (`r_ID`),
   UNIQUE KEY `r_PRACTID` (`r_PRACTID`,`r_pid`)
 ) ENGINE=InnoDB;
+
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `form_eye_base`
+--
+
+CREATE TABLE IF NOT EXISTS `form_eye_base` (
```

Comment:
For these new tables, use the standard:
DROP TABLE IF EXISTS `<table-name>`;
CREATE TABLE `<table-name>`;

Reply:
For these new tables, use the standard:
DROP TABLE IF EXISTS `<table-name>`;
CREATE TABLE `<table-name>`;